lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -676,160 +733,169 @@ defmodule Kernel.ErrorsTest do
       "nofile:1: invalid argument for alias, expected a compile time atom or alias, got: 42",
       'alias IO.{ANSI, 42}'
+    )
 
-    assert_eval_raise CompileError,
+    assert_eval_raise(
+      CompileError,
       "nofile:1: :as option is not supported by multi-alias call",
       'alias Elixir.{Map}, as: Dict'
+    )
 
-    assert_eval_raise UndefinedFunctionError,
+    assert_eval_raise(
+      UndefinedFunctionError,
       "function List.\"{}\"/1 is undefined or private",
       '[List.{Chars}, "one"]'
+    )
   end
 
   test "macros error stacktrace" do
-    assert [{:erlang, :+, [1, :foo], _},
-            {Kernel.ErrorsTest.MacrosErrorStacktrace, :sample, 1, _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosErrorStacktrace do
-        defmacro sample(num), do: num + :foo
-        def other, do: sample(1)
-      end
-      """)
+    assert [
+             {:erlang, :+, [1, :foo], _},
+             {Kernel.ErrorsTest.MacrosErrorStacktrace, :sample, 1, _} | _
+           ] =
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosErrorStacktrace do
+               defmacro sample(num), do: num + :foo
+               def other, do: sample(1)
+             end
+             """)
   end
 
   test "macros function clause stacktrace" do
     assert [{__MODULE__, :sample, 1, _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosFunctionClauseStacktrace do
-        import Kernel.ErrorsTest
-        sample(1)
-      end
-      """)
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosFunctionClauseStacktrace do
+               import Kernel.ErrorsTest
+               sample(1)
+             end
+             """)
   end
 
   test "macros interpreted function clause stacktrace" do
     assert [{Kernel.ErrorsTest.MacrosInterpretedFunctionClauseStacktrace, :sample, 1, _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosInterpretedFunctionClauseStacktrace do
-        defmacro sample(0), do: 0
-        def other, do: sample(1)
-      end
-      """)
+             rescue_stacktrace("""
+             defmodule Kernel.ErrorsTest.MacrosInterpretedFunctionClauseStacktrace do
+               defmacro sample(0), do: 0
+               def other, do: sample(1)
+             end
+             """)
   end
 
   test "macros compiled callback" do
-    assert [{Kernel.ErrorsTest, :__before_compile__, [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}], _} | _] =
-      rescue_stacktrace("""
-      defmodule Kernel.ErrorsTest.MacrosCompiledCallback do
-        Module.put_attribute(__MODULE__, :before_compile, Kernel.ErrorsTest)
-      end
-      """)
+    assert [
+             {
+               Kernel.ErrorsTest,
+               :__before_compile__,
+               [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}],
+               _
+             }
+             | _
+           ] =
```

Comment:
Same here, please break the stacktrace match apart.

Reply:
Am I on the right track with this one?
```
    assert [callback | _] =
             rescue_stacktrace("""
             defmodule Kernel.ErrorsTest.MacrosCompiledCallback do
               Module.put_attribute(__MODULE__, :before_compile, Kernel.ErrorsTest)
             end
             """)

    assert {
             Kernel.ErrorsTest,
             :__before_compile__,
             [%Macro.Env{module: Kernel.ErrorsTest.MacrosCompiledCallback}],
             _
           } = callback
```